test/password.cpp

```diff
@@ -0,0 +1,22 @@
+#include "catch.hpp"
+
+#include <password.h>
+
+using namespace newsbeuter;
+
+TEST_CASE("read_password()", "[password]") {
+    REQUIRE(read_password("data/empty.xml") == "");
+    REQUIRE(read_password("data/not-existing.txt") == "");
+    REQUIRE(read_password("data/single-line-string.txt") == "single line with spaces");
+    REQUIRE(read_password("data/multi-line-string.txt") == "string with spaces");
+}
+
+TEST_CASE("eval_password()", "[password]") {
```

Comment:
See the comment above. In this case, the description might say "eval_password() returns the first line of command's output".